apps/payment/signals.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from .models import FikenSale
+from .serializers import FikenSaleSerializer
+from .tasks import register_sale_with_fiken
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(post_save, sender=FikenSale)
+def handle_fiken_sale_created(sender, instance: FikenSale, created=False, **kwargs):
+    if created:
+        logger.info(f'Registering {instance} with Fiken')
+        sale_data = FikenSaleSerializer(instance).data
+        register_sale_with_fiken.delay(sale_data=sale_data)
+    else:
+        logger.warning(f'{instance} updated after creation')
```

Comment:
Should this be denied? If so, do that at the model layer, rather than having out-of-sync items in OW4 and Fiken.

If this should be supported in the long run, disable it for now and tell someone to contact dotkom/bankom to fix it for now, and tell them that this functionality is not supported yet.

Reply:
Soo, maybe. The way I thought about it is that nothing should be changed after it has been created. If I understand it correctly a change should be first removed and created again if something like this happens

Reply:
I'd just deny it on the model level for now (like, make the whole model non-editable, at least for non-superusers), and get a better understanding of how this is supposed to work in the long run. Should it be deleted and re-created? Should it be updated in-place? And fix it at a later time :)

Reply:
After having a chat with the correct people; it should not be possible to edit. I can't find any other way to do that than marking all fields as `editable=False`?